pkg/controller/che/che_controller.go

```diff
@@ -92,6 +94,12 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		if err := userv1.AddToScheme(mgr.GetScheme()); err != nil {
 			logrus.Errorf("Failed to add OpenShift User to scheme: %s", err)
 		}
+        if err := consolev1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
TBH I would have preferred this to be done *only when the console link is supported by the underlying OpenShift cluster* (simply inside a condition calling the `hasConsolelinkObject` function), even though I agree we might have noticed that unknown schemas/objects are ignored when unknown. Still seems cleaner (and clearer) to me.

@sleshchenko wdyt ?

Reply:
For me adding types to schema it's something as `init` operation, and `add` seems to be such a place.
Moving it to a condition calling the `hasConsolelinkObject` would mean that we register ( or try to register since we can add the corresponding `isAlreadyRegistered` method before ) each time when the operator is triggered (of course if consolelink is supported).
Personally I would leave as is to just configure client in one place.

But if you have a strong opinion that it must be done in the condition - it's acceptable for me, so I'll do.

Reply:
Oh, I wasn't clear.
I was meaning let it here, but just as add a condition, just similarly as it is done in line [87](https://github.com/eclipse/che-operator/pull/71/files#diff-d9445e614f0d728fd140771b0fd29c50R87) to restrict these new registrations to the case when console links are supported.

Reply:
> I was meaning let it here, but just as add a condition, just similarly as it is done in line 87 to restrict these new registrations to the case when console links are supported.

Ah, sorry for the misunderstanding. 
Then it makes sense.